river/tree/mondrian/mondrian_tree.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC
+from river.tree.mondrian.mondrian_tree_nodes import MondrianTreeBranch
+
+
+class MondrianTree(ABC):
+    """
+    Base class for Mondrian Trees.
+
+    This is an **abstract class**, so it cannot be used directly. It defines base operations
+    and properties that all the Mondrian Trees must inherit or implement according to
+    their own design.
+
+    Parameters
+    ----------
+    n_features
+        Number of features
+    step
+        Step parameter of the tree
+    loss
+        Loss to minimize for each node of the tree
```

Comment:
Are there other options already available?

Reply:
Not yet for the Classifier. I haven't checked yet for the Regressor, I'm just keeping it here in case the Regressor has different loss support (see MondrianTreeClassifier it doesn't have the loss parameter)